pkg/informer/supportedResources.go

```diff
@@ -86,12 +86,48 @@ func isResourceAllowed(group, kind string, allowedList []Resource, deniedList []
 				}
 			}
 		}
+		//case where same resources are in deny and allow list. expected behavior is to exclude those resources (default to deny) but we need to add a warning case.
+
+		allowMap := GetAllowDenyDataMap(allowedList)
+		denyMap := GetAllowDenyDataMap(deniedList)
+
+		for _, allow := range allowMap {
+			for _, deny := range denyMap {
+				if (allow["group"] == deny["group"] && allow["rec"] == deny["rec"]) || (allow["group"] == "*" && deny["group"] == "*" && allow["rec"] == deny["rec"]) ||
+					(allow["rec"] == "*" && deny["rec"] == "*" && allow["group"] == deny["group"]) {
+					glog.V(2).Infof("Deny Resource [group: '%s' kind: %s]. Resource present in both allow and deny rule.", group, kind)
+					return false
+				}
+			}
+		}
```

Comment:
This block of code is unreachable because this function returns when it matches the resource in the deny list above.

Reply:
ah, you are right! will refactor so it is included in both lists.